wg-notebooks/README.md

```diff
@@ -0,0 +1,37 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://github.com/kubeflow/community/generator/README.md
+--->
+# Notebooks Working Group
+
+WG Notebooks is responsible for the user experience around Notebooks and other web IDEs and their integrations with Kubeflow
```

Comment:
Do we need to regenerate this based on the latest changes to wgs.yaml?

Reply:
good catch!
I tried running the `make generate` but now that the Feature Store SIG is merged I'm getting a `ERROR: sig-feature-store: has no subprojects`. I'll create a corresponding issue for this.

Reply:
Created #415